src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThread.java

```diff
@@ -146,10 +157,33 @@ public StepExecution getStep() {
         this.step = step;
     }
 
+    /** TODO pending full coverage in {@link CpsDefaultGroovyMethods} */
```

Comment:
Such as is added in https://github.com/cloudbees/groovy-cps/pull/52, but that mixes in other stuff which we do not need.